package.json

```diff
@@ -22,13 +26,19 @@
   },
   "homepage": "https://github.com/andela/codepirates-ah-frontend#readme",
   "devDependencies": {
+    "cypress": "^3.4.1",
     "eslint": "^6.4.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-flowtype": "^3.13.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-react-hooks": "^1.7.0"
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "jest": "^24.9.0",
+    "node-sass": "^4.12.0",
```

Comment:
node-sass is supposed to be in dependencies

Reply:
Fixed